services/madoc-ts/src/frontend/shared/atoms/Heading3.tsx

```diff
@@ -11,6 +11,13 @@ export const Heading3 = styled.h3<{ $margin?: boolean }>`
     css`
       margin-bottom: 1em;
     `}
+    
+  & > a {
+    transition: color 0.3s;
+    &:hover {
+      color: #3579f6;
```

Comment:
I went for the same blue here as the primary buttons - not sure if this is theme-dependent though?